.github/workflows/release.yml

```diff
@@ -115,6 +122,8 @@ jobs:
         if: success()
         uses: micronaut-projects/github-pages-deploy-action@grails
         env:
+          SITEMESH_MAVEN_PASSWORD: ${{ secrets.GITHUB_TOKEN }}
+          CHECK_SITEMESH_REPO: true
```

Comment:
Are these necessary here?

Reply:
The docs will generate and need to pull depednencies, so I assume they'll need to pull the sitemesh dependency.  In which case it's needed.